src/api/dataSource/NodeDataSource.ts

```diff
@@ -407,9 +407,10 @@ export class NodeDataSource implements NodeApi {
         await this.client.post<InstallApplicationResponse>(
           `${getAppEndpointKey()}/admin-api/install-application`,
           {
-            contract_app_id: selectedPackageId,
-            version: selectedVersion,
             url: ipfsPath,
+            version: selectedVersion,
+            // TODO: parse hash to format
```

Comment:
Why is this required?

Reply:
check is downloaded hash is the same as saved in package manager. Preventing installation of corrupt or changed version